torch/csrc/jit/interpreter.cpp

```diff
@@ -126,6 +127,40 @@ void desugarTripCounts(Block * b) {
   }
 }
 
+// Currently inline the Fork node's block to evaluate it eagerly.
```

Comment:
Rather than desugaring the nodes, we should just implement fork/join as operators (in `register_prim_ops.cpp`).